SuperBuild/External_aruco.cmake

```diff
@@ -0,0 +1,79 @@
+IF(aruco_DIR)
+  FIND_PACKAGE(aruco 2.0.19 REQUIRED NO_MODULE)
```

Comment:
I have only tested with 2.0.19, there was a bug fix for linking under MSVC in version 2.0.16.